src/util/method.ts

```diff
@@ -0,0 +1,111 @@
+/**
+ * @ignore
+ */ /** */
+
+import Metadata from '@polkadot/metadata';
+import { createType, createTypeUnsafe, TypeRegistry } from '@polkadot/types';
+import { Call } from '@polkadot/types/interfaces';
+import { AnyJson } from '@polkadot/types/types';
+
+import { EXTRINSIC_VERSION, ONE_SECOND } from './constants';
+import { BaseTxInfo, UnsignedTransaction } from './types';
+
+export type Args = Record<string, AnyJson>;
+
+export interface Method {
```

Comment:
@joepetrowski This is basically the `methodData: any` you had before, I put some more structure in it